docs/docs/guide/02-hello.md

```diff
@@ -238,54 +231,22 @@ In the `query.proto` file, the response accepts `text`.
 - Use a text editor to modify the `x/hello/keeper/grpc_query_hello.go` file that contains the keeper function.
 - On the last line of the keeper function, change the line to return "Hello, Ignite CLI!":
 
-```go
+```go title="x/hello/keeper/grpc_query_hello.go"
 func (k Keeper) Hello(goCtx context.Context, req *types.QueryHelloRequest) (*types.QueryHelloResponse, error) {
- if req == nil {
-  return nil, status.Error(codes.InvalidArgument, "invalid request")
- }
- ctx := sdk.UnwrapSDKContext(goCtx)
- _ = ctx
- return &types.QueryHelloResponse{Text: "Hello, Ignite CLI!"}, nil // <--
+  if req == nil {
+    return nil, status.Error(codes.InvalidArgument, "invalid request")
+  }
+  ctx := sdk.UnwrapSDKContext(goCtx)
+  _ = ctx
+  // remove-next-line
+  return &types.QueryHelloResponse{}, nil
+  // highlight-next-line
+  return &types.QueryHelloResponse{Text: "Hello, Ignite CLI!"}, nil
```

Comment:
😃 👍

Reply:
Thanks @tbruyelle for the tip on using these comments 👍